apps/labs/posts/nebari-typer-cli.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'The evolution of Nebari CLI using Typer'
+published: October 4, 2022
+author: asmi-jafar
+description: 'Nebari typer CLI'
+featuredImage:
+  src: /posts/asmi-blogposts/nebari-logo.png
+  alt: 'An image of the assigned mentors.'
+hero:
+  imageSrc: /posts/asmi-blogposts/nebari-logo.png
+  imageAlt: 'Nebari theme logo.'
+---
+
+Hey, I'm [Asmi Jafar](https://twitter.com/asmijafar20), a computer science postgraduate student from India.
+I've worked on the [Nebari project](https://www.nebari.dev/) (previously known as Qhub) during my three months internship at Quansight Labs.
+
+## 🪴 About Nebari
+
+Nebari is a open source project from Quansight that enables teams/user to build cost-effective and scalable compute/data science platform in the cloud. It uses an infrastructure-as-code approach to quickly and easily deploy a shared data science environment, on premises or in the cloud of choice.
+
+## 👩‍💻 What is Command-line Interface
+
+CLI is a structure of the application through which user can interact and able to run that application or a program. It connects a user to a program.
+
+## Nebari CLI (Command-line Interface)
+
+I've worked on developing the Nebari CLI during my three months internship.
+
+Nebari CLI consists of various commands that user needs to run in order to use the Nebari. Let's see briefly what each one of the `nebari` commands does.
+
+Below is the diagram of the Nebari CLI structure.
+
+<img alt="Nebari commands" src="/posts/asmi-blogposts/nebari-commands.png" />
```

Comment:
@pavithraes and I iterated on this a bit [here](https://docs.google.com/presentation/d/1lSeydRDhhqZ9mhZHtOJlKSrmpIWCBaKyLU9GyRxq1So/edit#slide=id.g15b6aade2db_1_78). 

It's not perfect, but I think it highlights the most important commands and what the result of each command is.

Reply:
<img width="1099" alt="Updated image" src="https://user-images.githubusercontent.com/6564007/199562342-0b3d66b8-e0ac-428e-bcf2-4afb7ac4cc2b.png">
Thanks for sharing the file @asmijafar20! I tweaked this image a bit because I noticed it was pixelated. @trallard would you like me to make additional changes? My main concerns were the nebari logo's proportions and crispness, but I retouched the arrow and cloud position while I was at it. I can undo it, or make other changes as you prefer.

@asmijafar20 If you would like to share the files you use to generate the other images/designs where the nebari logo appears (hero and feature images), I can repeat the same tweaks. If Tania confirms it, then I believe those are the last changes 😇

Reply:
I have uploaded the hero and feature images in the drive link of the template images.

Reply:
It's not in that folder now 😿  can you share it with me again?

Reply:
I don't have access to the drive folder now. What to do now?